frontend/src/utils/serializeFilterParams.js

```diff
@@ -38,23 +42,25 @@ export default function serializeFilterParams(filters, descriptions) {
 
       case FILTER_TYPE.INPUT: {
         const options = filters[field].options
-
-        if (options) {
-          if (options.recursiveMatch)
-            key += "__recursive"
-          else if (options.exactMatch)
-            key += "__startswith"
-          else
-            key += "__icontains"
-        } else {
-          key += "__icontains"
-        }
-
-        if(value)
-          params[key] = value
-
-        break;
-      }
+        const isBatch = description.batch && hasWhiteSpace(value)
+
+        if(!isBatch){
+          if (options) {
+            if (options.recursiveMatch)
+                key += "__recursive"
+            else if (options.exactMatch)
+                key += "__startswith"
+            else
+                key += "__icontains"
+        } else{
+           key += "__icontains"
```

Comment:
I don't see icontains or containts as a good batch comparison. Most likely this will be used to extensively list all desired barcodes or names not an approximation of each (this could be ... i'm just not sure this would be the expected result)

Reply:
I agree with Ulysse, if we do a batch search, we expect an exact match. Also, ideally, it should be reflected in the interface that the search is 'exact' when we do a batch search.